[lit] Fix the timeout.

JDevlieghere opened the pull request:
The timeout wasn't working because it's a property of the lit
configuration, not of the configuration in lit.site.cfg. This sets the
property for the correct object.

git-svn-id: https://llvm.org/svn/llvm-project/lldb/trunk@359492 91177308-0d34-0410-b5e6-96231b3b80d8
(cherry picked from commit febdfc325332e113c9b400e1fd943a13366a25f1)

JDevlieghere committed:
[lit] Fix the timeout.

The timeout wasn't working because it's a property of the lit
configuration, not of the configuration in lit.site.cfg. This sets the
property for the correct object.

git-svn-id: https://llvm.org/svn/llvm-project/lldb/trunk@359492 91177308-0d34-0410-b5e6-96231b3b80d8
(cherry picked from commit febdfc325332e113c9b400e1fd943a13366a25f1)

adrian-prantl reviewed (approved):
LGTM, if this is a swift-lldb change. Otherwise this should go into llvm.org.

JDevlieghere commented:
@adrian-prantl It's a cherry-pick.

@swift-ci please test and merge

JDevlieghere commented:
@swift-ci please test and merge

JDevlieghere committed:
[lit] Check for the psutil module when setting a timeout

Apparently setting the per-test-timeout and not having the psutil
package constitutes to a fatal error. So only set the timeout when the
module is available.

git-svn-id: https://llvm.org/svn/llvm-project/lldb/trunk@359503 91177308-0d34-0410-b5e6-96231b3b80d8
(cherry picked from commit a2ecd22efcb1548a4d3eaea2fda2ffcf1384b1ce)
